templates/web/base/report/update.html

```diff
@@ -24,7 +24,24 @@ <h4 class="static-with-rule">[% loc('Updates') %]</h4>
             <div class="item-list__update-wrap">
             [% IF update.whenanswered %]
                 <div class="item-list__update-text">
-                    <p class="meta-2"> [% INCLUDE meta_line %] </p>
+                    <p class="meta-2">
+                        [% INCLUDE meta_line %]
```

Comment:
This section is not necessary, there is no name shown for a questionnaire response (when update.whenanswered is true).